aleph-client/Cargo.toml

```diff
@@ -1,13 +1,14 @@
 [package]
 name = "aleph_client"
-version = "0.4.1"
+version = "0.4.2"
 edition = "2021"
 
 
 [dependencies]
 # Substrate dependencies
 sp-core = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", features = ["full_crypto"] }
 sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", default-features = false }
+sp-io = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13" }
```

Comment:
is aleph-client ever used in runtime (no-std environment?) if not, maybe we can get rid of the std features ?

Reply:
It's only used in native I think. How exactly we can get rid of them?

Reply:
get rid of the features section in Cargo.toml and add features = ["std"] for all substarte deps - that should do it I suppose

Reply:
Right, but in the end thanks to @kostekIV suggestion `sp-io` was not neeeded

Reply:
Could you still create a ticket in jira for that cleanup so that we don't forget?